rollup/docker-compose-proposer-tool.yml

```diff
@@ -0,0 +1,50 @@
+version: '3'
+
+services:
+  db:
+    image: postgres:14
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_DB=scroll
+    ports:
+      - "5432:5432"
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U postgres"]
+      interval: 5s
+      timeout: 5s
+      retries: 5
+
+  db-client:
```

Comment:
So you always want to reset the DB of the proposer tool upon restart? if so then you could also just call it in the code no?

Reply:
Makes sense. removed in [0836a1d](https://github.com/scroll-tech/scroll/pull/1645/commits/0836a1d78f357fc595fcbdee93db658a2811303e).